code/model/ForumThread.php

```diff
@@ -133,7 +133,15 @@ function getFirstPost() {
 	 * @return int
 	 */
 	function getNumPosts() {
-		return (int)DB::query("SELECT count(*) FROM \"Post\" WHERE \"ThreadID\" = $this->ID")->value();
+		return (int)DB::query(
```

Comment:
We should double quote (ansi sql standard) all SQL identifiers. `Post.AuthorID` should be `"Post"."AuthorID"`. this helps cross-db connector compatibility. :D